### PR TITLE
Fix maintainers url

### PR DESCRIPTION
**Pull request by bgeuken**

### CODE REVIEW

**Comment by bgeuken** [comment]
@ChrisBr Please review. I've squashed the two commits from Manuel as requested and updated the css. The last commit is temporary and I'll squash it into the first one, if you approve the changes.

**Comment by ChrisBr** [comment]
LGTM, go for it!
Thanks :+1:

**Comment by coolo** [comment]
this seems to be deployed now. But the closing of dialogs doesn't seem to be tested well enough.

At least I can now open the dialog on https://build.opensuse.org/request/show/446574, but not close it.

File: src/api/app/assets/javascripts/webui/application.js [hunk 1]
```diff
@@ -246,3 +246,13 @@ function select_from_autocomplete(toselect) {
         if ($(this).text() == toselect) { $(this).trigger('mouseenter').click(); }
     });
 }
+
+$(function() {
+  $('.show_dialog').on('click', function() {
+    $($(this).data('target')).removeClass('hidden');
+  });
+
+  $('.close-dialog').on('click', function() {
```
Comment (coolo) [inline src/api/app/assets/javascripts/webui/application.js]: beginner's error: you can't bind elements that aren't there. I suggest you stop merging refactorings without test :(

**Pull request merged**